.github/ISSUE_TEMPLATE.md

```diff
@@ -23,6 +25,10 @@ Place an `x` between the square brackets on the lines below for every satisified
 <!--
 Please add a series of steps to reproduce the problem. See https://stackoverflow.com/help/mcve for in depth information
 on how to create a mimimal, complete, and verifiable example.
+
+TL;DR version: If we get a report like: "I use `library/tool X` and here's the one line of code that the error points to," we can provide troubleshooting suggestions but we generally can't know that if we use `library/tool X` we'll be doing the same thing that causes your issue. *Ideally* we would like to be able to copy-paste the code contents of your report (the smaller the better) into an empty directory and run `npm i mocha <and any other packages you specify>` followed by `mocha <arguments you specified>` (and/or any other commands necessary) and see the problem occur.
+
+*That doesn't mean we won't field issues that can't be so easily reproduced*, and we don't expect you to dig into Mocha specifically (we know not everyone is as familiar with it, especially some of the nuances under the hood). However, if it's possible to create such a case it will often narrow down the problem from "could be almost anything" to one of a few (or even just one) specific thing(s)! Isolating the problem like that is something we can't always do without lots of info from you (info that a reproducible test case can comprehensively provide -- see the link above for more on how and why), but *is a crucial step in getting you a solution*. Plus, creating such test cases is a great exercise in troubleshooting even outside of issue reports, so it helps you personally too!
```

Comment:
I think we can probably put this type of thing in the wiki or whatever for now instead of the issue template.  I'd like to ideally _remove stuff_ from the template since keeping it short is better DX, IMO

Reply:
As it is I'm not sure everyone is following and reading the link to the detailed explanation/rationale (from StackExchange) for MCVEs. I was hoping to give an *alternative* that's available right here, right now and briefer -- without leaving out any of the necessary steps and without just demanding "Do X and Y and Z" without a reason (or some sympathy, since it is more involved for the user). I'd rather try to tweak it to streamline that than risk just replicating the "people appear not to be reading the StackExchange link" problem with our own wiki, if possible.

As long as the only feature GitHub gives us is to put in a template full of *information*, we are kind of between a rock and a hard place getting people to pay attention to all the relevant information. If we could actually place requirements on fields to fill out that would be validated before the issue is filed, we could solve this, but as-is the best we can hope for is a balance between "too much information in the template" and "too much information outside the template". Unfortunately I suspect the users who really need the information spelled out for them are also the users most sensitive to that balance...

Reply:
How about the following simplification?

> Ideally, include minimal code that we can copy-paste into an empty project and run to see the issue. Not every issue has to meet that ideal, but for those that can, it ensures we have all the info needed to look for a solution to your problem -- and you may even solve it yourself while trying to narrow the code down!